.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+#### Checklist
+
+- [ ] Chart Version bumped
+
+
```

Comment:
added a pull request template.
https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates#pull-request-templates